File: sdks/java/lib/src/main/java/com/hooksniff/models/EnvironmentModelOut.java
package com.hooksniff.models;

import com.fasterxml.jackson.annotation.JsonProperty;

public class EnvironmentModelOut {
    public String id;
    @JsonProperty("customer_id") public String customerId;
    public String name;
    public String slug;
    public String description;
    @JsonProperty("is_default") public boolean isDefault;
    public String color;
    @JsonProperty("created_at") public String createdAt;
    @JsonProperty("updated_at") public String updatedAt;
    @JsonProperty("variable_count") public Long variableCount;
}

File: sdks/go/models/environment_model_out.go
package models

import "time"

type EnvironmentModelOut struct {
	Id             string     `json:"id"`
	CustomerId     string     `json:"customerId"`
	Name           string     `json:"name"`
	Slug           string     `json:"slug"`
	Description    *string    `json:"description,omitempty"`
	IsDefault      bool       `json:"isDefault"`
	Color          *string    `json:"color,omitempty"`
	CreatedAt      time.Time  `json:"createdAt"`
	UpdatedAt      time.Time  `json:"updatedAt"`
	VariableCount  *int64     `json:"variableCount,omitempty"`
}
